homeassistant/components/tessie/entity.py

```diff
@@ -0,0 +1,45 @@
+"""Tessie parent entity class."""
+
+
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, MODELS
+from .coordinator import TessieDataUpdateCoordinator
+
+
+class TessieEntity(CoordinatorEntity[TessieDataUpdateCoordinator]):
+    """Parent class for Tessie Entities."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: TessieDataUpdateCoordinator,
+        key: str,
+    ) -> None:
+        """Initialize common aspects of a Tessie entity."""
+        super().__init__(coordinator)
+        self.vin: str = coordinator.vin
```

Comment:
```suggestion
        self.vin = coordinator.vin
```

already typed at the source